src/restapi/manage.rs

```diff
@@ -0,0 +1,110 @@
+use actix_web::{http, web, Responder};
+//use web::Json;
+use futures::future::OptionFuture;
+
+use super::{state, types};
+use crate::matchengine::server::matchengine::{self, matchengine_client::MatchengineClient};
+use crate::storage;
+
+pub mod market {
+
+    use super::*;
+
+    pub async fn add_assets(req: web::Json<types::NewAssetReq>, app_state: web::Data<state::AppState>) -> impl Responder {
+        let assets_req = req.into_inner();
+
+        for asset in &assets_req.assets {
+            log::debug!("Add asset {:?}", asset);
+            if let Err(e) = storage::config::persist_asset_to_db(&app_state.db, asset, assets_req.force_update).await {
+                return (e.to_string(), http::StatusCode::INTERNAL_SERVER_ERROR);
+            }
+        }
+
```

Comment:
in "add_pair" we may reload (depending on the request)

why in "add_assets" we don't reload at all?

Reply:
I had thought it is no need to reload after just adding assets because it bring nothing change to the running matchengine. Now I found we may need to simply depositing onto an assets (even it is not traded in any market). So I will add reloading option on it.